Features/Powerbi_integration_exports/testPages/data/power_bi_page.py

```diff
@@ -0,0 +1,275 @@
+
+
+""""Contains test page elements and functions related to the Lookup Table module"""
+
+class PowerBiPage(BasePage):
+
+    def __init__(self, driver):
+        super().__init__(driver)
+        self.power_bi = (By.XPATH, "//*[@id='hq-sidebar']/nav/ul[1]/li[7]/a")
+        self.add_odata_feed = (By.XPATH, "//*[@id='create-export']/p/a")
+        self.disabled = (By.XPATH,"//*[@data-bind='visible: showSubmit, disable: disableSubmit']")
+        self.feed_type = (By.XPATH,"//select[@id='id_model_type']")
+        self.feed_type_visible = (By.XPATH,"//*[@data-bind='visible: modelType()']")
+        self.case_type = (By.XPATH,"//*[@for='id_case_type']")
+        self.cancel_button = (By.XPATH,"//*[@id='createExportOptionsModal']/div/form/div/div[7]/button[1]")
+        self.applications = (By.XPATH,"//select[contains(@name,'application')]")
+        self.app_dropdown = (By.XPATH,"//input[contains(@aria-controls,'id_application-results')]")
+        self.users_list_item = "//ul[@role='listbox']/li[contains(.,'{}')]"
+        self.menu =(By.XPATH,"//select[contains(@name,'module')]")
+        self.menu_select = (By.XPATH,"//input[@aria-controls='select2-id_module-results']")
+        self.form = (By.XPATH,"//select[contains(@placeholder,'Select Form')]")
+        self.add_export_conf = (By.XPATH, "//button[@data-bind='visible: showSubmit, disable: disableSubmit']")
+        self.submission_msg = (By.XPATH, "//span[contains(@data-bind,'text: submissionCountText')]")
+        self.case = (By.XPATH,"//span[@aria-controls='select2-id_case_type-container']")
+        self.case_select = (By.XPATH,"//input[@aria-controls='select2-id_case_type-results']")
+        self.save = (By.XPATH,"//button[@type='submit'][contains(@data-bind,'click: save')]")
+        self.settings = (By.XPATH, "//*[@id='customize-export']/header/div/div/h3")
+        self.success = (By.XPATH,"//*[@class='alert alert-margin-top fade in alert-success']")
+        self.odata_feed_name = (By.XPATH,"//input[@id='export-name']")
+        self.powerBI_tab_int = (By.LINK_TEXT, "PowerBi/Tableau Integration")
+        self.select_all_btn = (By.XPATH, '//button[@data-bind="click: selectAll"]')
+        self.delete_selected_exports = (By.XPATH, '//a[@href= "#bulk-delete-export-modal"]')
+        self.bulk_delete_confirmation_btn = (By.XPATH, '//button[@data-bind="click: BulkExportDelete"]')
+        self.show_advance = (By.XPATH, "//span[@data-bind='visible: !table.showAdvanced()']")
+        self.hide_advance = (By.XPATH,"//span[@data-bind='visible: table.showAdvanced']")
+        self.show_delete = (By.XPATH,"//span[@data-bind='visible: !table.showDeleted()']")
+        self.copy_edit_button = (By.XPATH,"(//*[@data-bind='if: isOData()'])[1]")
+        self.description = (By.XPATH,("//*[@id='export-description']"))
+        self.disabled1 =(By.XPATH,("//td[5][./input[@disabled]]//preceding-sibling::td//input[@type='checkbox' and @disabled='disabled']"))
```

Comment:
please don't name variables and functions with 1/2/and so on. provide proper names, either new or old or before or after.